hazelcast-integration/spring-data-hazelcast-chemistry-sample/common/src/main/java/com/hazelcast/samples/spring/data/chemistry/CommonConfiguration.java

```diff
@@ -14,32 +10,4 @@
 @Configuration
 @EnableHazelcastRepositories(basePackages = {"com.hazelcast.samples.spring.data.chemistry"})
```

Comment:
I see the same annotation `@EnableHazelcastRepositories` in `AfterTranslatorConfiguration.java` too. Do we really need to define this two times?

Reply:
You're right. Thanks Mesut!
I've removed the annotation from `AferTranslatorConfiguration.java`.